updater.py

```diff
@@ -17,19 +18,21 @@
 parser.add_argument("-v", "--verbose", help="Verbose", action="store_true")
 args = parser.parse_args()
 
+if args.verbose:
+    logging.basicConfig(format='%(asctime) s%(levelname)s:%(message)s', level=logging.DEBUG, datefmt='%m/%d/%Y %I:%M:%S')
```

Comment:
Testing comment, this if statement could be simplified with the logger defaults being set then just adjusting the log level in the if statement. Reduces the risk of deviation in log format between different log levels